docs/source/usersguide/input.rst

```diff
@@ -1116,6 +1116,15 @@ Each ``material`` element can have the following attributes or sub-elements:
     .. note:: If one nuclide is specified in atom percent, all others must also
               be given in atom percent. The same applies for weight percentages.
 
+    An optional attribute/sub-element for each nuclide is ``scattering``. This
+    attribute may be set to "ace" to use the scattering laws specified in the
+    ACE files (default). Alternatively, when set to "lab", the ACE scattering
```

Comment:
I think "isotropic" might be a bit more obvious than "lab". I also don't really like "ace" because there will probably be a day when we no longer use ACE cross sections, but not sure what a better identifier is.